wntr/tests/test_msx_elements.py

```diff
@@ -176,13 +176,14 @@ def test_MsxModel_creation_specific_everything(self):
         equil2 = rxn_model1.add_reaction("Cl", "pipe", "equil", "-Ka + Kb * Cl + T0")
         rate2 = rxn_model1.add_reaction("Cl", "tank", wntr.msx.ExpressionType.R, "-Ka + Kb * Cl + T0", note="Foo Bar")
         formula2 = rxn_model1.add_reaction("ClOH", "PIPE", "formula", "-Ka + Kb * Cl + T0")
-        rxn_model1.add_source('Cl', 'Dummy1', 1.23)
-        rxn_model1.add_source('Cl', 'Dummy2', 2.7)
+        rxn_model1.add_source("CONCEN", 'Cl', 'Dummy1', 1.23, note="foo")
+        rxn_model1.add_source("SETPOINT", 'Cl', 'Dummy2', 2.7)
         source = rxn_model1.get_sources_for_species('Cl')
-        self.assertDictEqual(source, {'Dummy1':1.23, 'Dummy2':2.7})
+        self.assertDictEqual(source, {'Dummy1':{'strength':1.23, 'source_type': 'CONCEN', 'pattern': None, 'note': 'foo'}, 
+                             'Dummy2':{'strength':2.7, 'source_type': 'SETPOINT', 'pattern': None, 'note': None}})
         rxn_model1.remove_source('Cl','Dummy1')
         source = rxn_model1.get_sources_for_species('Cl')
-        self.assertDictEqual(source, {'Dummy2':2.7})
+        self.assertDictEqual(source, {'Dummy2':{'strength':2.7, 'source_type': 'CONCEN', 'pattern': None, 'note': None}})
```

Comment:
This needs to be ``"SETPOINT"`` to match the inputs above - probably easiest if you make that change so that it automatically updates the PR?